### PR TITLE
Update CHANGELOG

### PR DESCRIPTION
**Pull request by ManaSugi**
Update CHANGELOG to include all notable changes from v0.1.3.

Signed-off-by: Manabu Sugimoto <Manabu.Sugimoto@sony.com>

### CODE REVIEW

**Comment by ManaSugi** [comment]
After merging this, I'll create a PR to change `Cargo.toml` and `CHANGELOG` (change `Unreleased` -> `[0.2.0]` and create new `Unreleased` section) for releasing.

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,16 +7,58 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 ### Added
-- `ScmpArgCompare::new` is now a `const fn`.
+- `ScmpVersion::current()` as rustified replacement for `get_library_version()`.
+- `ScmpFilterContext::get_no_new_privs_bit()` to query the state of the No New Privileges bit.
+- `ScmpArch::native()` as rustified replacement for `get_native_arch()`.
+- `ScmpFilterContext::as_ptr()` to return a raw pointer to the `scmp_filter_ctx`.
+- `scmp_cmp!`macro to create a `ScmpArgCompare` in a more elegant way.
```
Comment (rusty-snake) [inline CHANGELOG.md:14]:
```suggestion
- `scmp_cmp!` macro to create a `ScmpArgCompare` in a more elegant way.
```

or

```suggestion
- `scmp_cmp!`-macro to create a `ScmpArgCompare` in a more elegant way.
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,16 +7,58 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
 ### Changed
+- `get_syscall_name_from_arch` and `get_syscall_from_name` output a syscall number with
+an error message when the functions cannot resolve the syscall name.
+- `ScmpAction::Trace` now holds an `u16` since you can not use any more bits anyway
+(**Incompatible change**).
 - `ScmpAction::Errno` now holds an `i32` to make `ScmpAction::Errno(libc::EPERM)`
-  work without casting.
-- `ScmpAction::Trace` now holds an `u16` since you can not use any more bits anyway.
+  work without casting (**Incompatible change**).
+- `ScmpFilterContext::add_rule()` returns an error if the length of comparators is bigger than `u32`.
```
Comment (rusty-snake) [inline CHANGELOG.md:35]:
```suggestion
```

After #66 not explicit via try_into but libseccomp will fail anyway if you want to compare more then 4294967295 arguments.

File: CHANGELOG.md [hunk 3]
```diff
@@ -7,16 +7,58 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+- `ScmpArgCompare::new` is now a `const fn`.
+- `ScmpFilterContext::export_{pfc,bpf}()` take all types which implement `AsRawFd`
+(**Incompatible change**).
```
Comment (rusty-snake) [inline CHANGELOG.md:38]:
```suggestion
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -7,16 +7,58 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+- `ScmpFilterContext::export_{pfc,bpf}()` take a `&mut` reference instead of consuming the ownership
+(**Incompatible change**).
+- `ScmpFilterContext::set_filter_attr()` takes `&mut self` rather than `&self` (**Incompatible change**).
+- Fixed memory leak in `get_syscall_name_from_arch`.
+- Made `ScmpArgCompare::new` to take only one `datum` (**Incompatible change**).
+- Made `ScmpCompareOp::MaskedEqual` to contain the mask (**Incompatible change**).
+- Implemented a debug trait for `SeccompError` by hand without the derive macro.
+- The `libseccomp-sys` crate supports the libseccomp library v2.5.3.
+- `ScmpFilterContext::add_rule()` does not take an `Option<&[ScmpArgCompare]>` argument
+(**Incompatible change**).
```
Comment (rusty-snake) [inline CHANGELOG.md:48]:
```suggestion
, use `add_rule_conditional` (**Incompatible change**).
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -7,16 +7,58 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+- `get_native_arch()` uses `ScmpArch::native()` instead.
 
 ### Removed
-- `Clone` on `ScmpFilterContext` because it cauesed double-free/use-after-free
-   in safe code.
+- `ScmpData` (was unused).
+- `.to_native()` functions (**Incompatible change**).
+- `Clone` on `ScmpFilterContext` because it causes double-free/use-after-free
+in the safe code (**Incompatible change**).
+- `code` field in `SeccompError`.
```
Comment (rusty-snake) [inline CHANGELOG.md:61]: Was it `pub`lic?

File: CHANGELOG.md (hunk 1 above)
Comment (ManaSugi) [reply CHANGELOG.md]: I fixed it.

File: CHANGELOG.md (hunk 2 above)
Comment (ManaSugi) [reply CHANGELOG.md]: Ok. Thank you. I removed it.

File: CHANGELOG.md (hunk 4 above)
Comment (ManaSugi) [reply CHANGELOG.md]: I fixed it.

File: CHANGELOG.md (hunk 3 above)
Comment (ManaSugi) [reply CHANGELOG.md]: I fixed it.

File: CHANGELOG.md (hunk 5 above)
Comment (ManaSugi) [reply CHANGELOG.md]: Oh, it was not `pub`, so I removed it.

**Review by rusty-snake — APPROVED** [review]

**Pull request merged**